natchez-extras-doobie/src/main/scala/com/ovoenergy/natchez/extras/doobie/TracedTransactor.scala

```diff
@@ -93,12 +93,24 @@ object TracedTransactor {
 
           override val executeQuery: TracedOp[ResultSet] =
             runTraced(super.executeQuery)
+
         }
 
       override lazy val ConnectionInterpreter: ConnectionInterpreter =
         new ConnectionInterpreter {
           override def prepareStatement(a: String): Kleisli[F, Connection, PreparedStatement] =
             super.prepareStatement(a).map(TracedStatement(_, a): PreparedStatement)
+          override def prepareStatement( a: String, b: Array[String]): Kleisli[F, Connection, PreparedStatement] =
+            super.prepareStatement(a, b).map(TracedStatement(_, a, b): PreparedStatement)
+          override def prepareStatement(a: String, b: Array[Int]): Kleisli[F, Connection, PreparedStatement] =
+            super.prepareStatement(a, b).map(TracedStatement(_, a, b): PreparedStatement)
+          override def prepareStatement(a: String, b: Int) =
+            super.prepareStatement(a, b).map(TracedStatement(_, a, b): PreparedStatement)
+          override def prepareStatement(a: String, b: Int, c: Int) =
+            super.prepareStatement(a, b, c).map(TracedStatement(_, a, b, c): PreparedStatement)
+          override def prepareStatement(a: String, b: Int, c: Int, d: Int) =
+            super.prepareStatement(a, b, c, d).map(TracedStatement(_, a, b, c, d): PreparedStatement)
```

Comment:
It seems the variadic parameter is not used anywhere, why is it needed to be set?